File: ts-sdk/src/sdk/models/shared/uploadurl.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// UploadUrl
/** 
 * The details for the upload session for the file.
**/
export class UploadUrl extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=upload_token" })
  uploadToken?: string;

  @SpeakeasyMetadata({ data: "json, name=upload_url" })
  uploadUrl?: string;
}

File: python-sdk/src/boxsdk/models/shared/uploadurl.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils


@dataclass_json
@dataclasses.dataclass
class UploadURL:
    r"""UploadURL
    The details for the upload session for the file.
    """
    
    upload_token: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('upload_token') }})
    upload_url: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('upload_url') }})
    
